addon/lib/templates.js

```diff
@@ -71,3 +71,24 @@ module.exports.feedback = `
   </div>
   <div class="clear"></div>
 </div>`;
+
+module.exports.installed = `
+<div id="browser-doorhanger">
+  <div class="copter-wrapper">
+    <div class="copter fly-down"></div>
+  </div>
+  <p>Test Pilot lets you test out experimental features in Firefox. Ready for takeoff?</p>
```

Comment:
Yeah, it's kind of annoying - and until @pdehaan noticed the issue, I'd mostly forgotten about L20n myself. :frowning: 

I wonder if there's any kind of validation or linting or something we can do to be sure we don't miss strings? I can't think of what it would be off the top of my head. Also doesn't help that we're not totally hooked up to a localizing interface to make this stuff actually useful